packages/readmodel-models/drizzle.config.ts

```diff
@@ -0,0 +1,28 @@
+import "dotenv/config";
+import { defineConfig } from "drizzle-kit";
+
+export default defineConfig({
+  out: "./src/drizzle",
+  dialect: "postgresql",
+  dbCredentials: {
+    host: process.env.READMODEL_SQL_DB_HOST!,
```

Comment:
We could avoid using no-null assertion here
```suggestion
import { ReadModelSQLDbConfig } from "pagopa-interop-commons";

const readmodelDBConfig = ReadModelSQLDbConfig.parse(process.env);

export default defineConfig({
  out: "./src/drizzle",
  dialect: "postgresql",
  dbCredentials: {
    host: readmodelDBConfig.readModelSQLDbHost,
```

Reply:
Fixed: now we avoid no-null assertions.